Thread: add HA Thread network when third-party Thread nw is present (#31405)
home-assistant/home-assistant.github.io: source/_integrations/thread.markdown

- Go to {% my integrations title="**Settings** > **Devices & Services**" %}.
- If you do not see a **Thread** integration, add it.
- Then, select **Configure** and check if you see any Thread networks on the overview page.
- Case 1: If you do not have any Thread networks yet, follow [Case 1: Make Home Assistant your first Thread network](#case-1-making-home-assistant-your-first-thread-network)
- Case 2: If you have existing networks, follow [Case 2: Create a HA border router when there is an existing network](#case-2-creating-a-ha-border-router-when-there-is-an-existing-network)

### Case 1: Making Home Assistant your first Thread network

> Follow these steps if you want to turn Home Assistant into a Thread border router using the Thread radio of Yellow, SkyConnect, or another compatible radio and you do not have any third-party Thread networks present yet.

Reviewer: ```suggestion
Follow these steps if you want to turn Home Assistant into a Thread border router using the Thread radio of Yellow, SkyConnect, or another compatible radio and you do not have any third-party Thread networks present yet. This will automatically create a new Thread network with the name `ha-thread-xxxx`. The last for digits are a network-specific identifier (PAN ID).
```

> Currently this use case requires an Android phone. If you want to add Matter devices via an iOS phone, this use case is not supported yet.

Reviewer: ```suggestion
Note: To add Matter devices to this Thread network, an Android phone is required. Adding Matter devices to this Thread network using an iOS phone is not yet supported.
```

   - Follow the instructions on screen.
   - **Result**: You will see a confirmation stating that Thread credentials from Home Assistant have been added to this device.

### Case 2: Creating a HA border router when there is an existing network

> Follow these steps if you want to turn Home Assistant into a Thread border router using the Thread radio of Yellow, SkyConnect, or another compatible radio but you already have third-party Thread networks present. These steps will join the two networks. For more information on why networks are joined, read the section on [Combining Thread networks](#combining-thread-networks).

Reviewer: ```suggestion
Follow these steps if you want to turn Home Assistant into a Thread border router using the Thread radio of Yellow, SkyConnect, or another compatible radio but you already have third-party Thread networks present. These steps will join the Home Assistant Thread border router with the existing Thread network.
```

![image](/images/integrations/thread/thread-no-preferred-network-but-third-party-present.png)

> If you have both Google and Apple border routers present, decide which one you would like to join with the Home Assistant border router.

Reviewer: ```suggestion
If you have both Google and Apple Thread networks present, decide which one you would like add the Home Assistant border router to.
```

#### Prerequisites

- Device with a Thread-capable radio, such as Home Assistant Yellow, SkyConnect, or another compatible radio
- Third-party Thread network listed

> - Android phone if you have a Google border router, iPhone if you have an Apple border router

Reviewer: ```suggestion
- Android phone if you have a Google Thread network, iPhone if you have an Apple Thread network
```

#### To create a HA border router when there is an existing network

> Note: the steps and images here are done with a Google border router. But the process is very similar if you are using an Apple border router with an iPhone.

Reviewer: ```suggestion
Note: the steps and images here show the process with a Google Thread network. But the process is very similar if you have an Apple Thread network with an iPhone.
```
